articles/other/other-ref-promon.md

```diff
@@ -0,0 +1,93 @@
+
+- Customer VDC, vApps, VMs, Virtual Network Appliance and customer networks - This is the customer's responsibility to monitor.
+
+- External connectivity, for example, internet, PSN, HSCN, Janet, RLI, leased lines - This is the responsibility of the telecommunications provider.
+
+<sup>*</sup> For more information about UKCloud's volumetric DDoS protection, see the [*DDoS mitigation service FAQs*](../connectivity/conn-faq-ddos.md).
+
+We do not share a feed from our platform protective monitoring to customers, so you cannot connect your own SOC to the protective monitoring data. You can however get access to log data for all traffic traversing your virtual boundary firewall.
+
+## Cyber Security News page
+
+The *Cyber Security News* page in the UKCloud Portal, located under Monitoring, lets you see, at a glance, general threat briefs and security news from around the world as analysed by e2e-assure.
+
+All Portal users can access the *Cyber Security News* page.
+
+Each brief includes a headline in bold text that sums up the threat to help you scan through the feed quickly. Briefs are also colour-coded, ranging from yellow to red, to indicate the severity of the brief: Yellow indicating a low threat level and red indicating a high threat level.
+
+A more in-depth description of the threat is included below the headline. This provides you with more information and recommended actions, depending on the threat. The feed is listed in chronological order with two timestamps: one showing when the brief was originally published and the other showing when the brief was most recently updated.
+
+You can search and filter the feed on the *Cyber Security News* page to define exactly what content you want to see. Use the search bar to query specific words or phrases within the threat briefs. To access filtering options, to edit the priority and date range you want to see in your feed or search results, click **Advanced Search**.
+
+## My Security Events page
+
+The *Security Events* page in the UKCloud Portal, located under Monitoring, lets you see, at a glance, threat information specific to your account.
+
+> [!NOTE]
+> To view the *Security Events* page, you must have **Read** monitoring permissions on the page. For more information, see [*Portal permissions*](../portal/ptl-ref-overview-permissions.md#permissions-for-monitoring).
+
+Currently, the *Security Events* page only includes information for internet connectivity to UKCloud for VMware in the Assured OFFICIAL security domain. All other services are supported by UKCloud's standard Protective Monitoring service. UKCloud's security team will inform you of security incidents for those other services in the usual way (that is, via the UKCloud Portal, email or phone call).
```

Comment:
Question for @LukeCJefferies  - presumably we raise a ticket in ITSM regardless of whether the info is also included in the My Security Events page in the Portal? If so I wonder if this text should be more along the lines of that we always inform via My Calls, and in addition if it is a VMware Assured alert then it can also be seen in the My Security Events section of the Portal?

Reply:
@cam-ukcloud, that is correct. 
As the Portal does not have a notification feature we have to raise the ticket to bring it to some ones attention.

Reply:
I've changed this paragraph and added some more info to address this